Update list of Orgs for BOD/Cybex reports

climber-girl opened the pull request:
As of January 2, 2020, OPIC is now the US International Development Finance Corporation (DFC).  They have new domains listed under "United States International Development Finance Corporation" on https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv, and I will be updating the OPIC request doc to reflect the new name/acronym.

Marina said she'll update the OPIC.gov agency name to "United States International Development Finance Corporation" this coming week.

climber-girl committed:
OPIC to DFC Transition

As of January 2, 2020, OPIC is not the US International Development Finance Corporation (DFC).  They have new domains listed under "United States International Development Finance Corporation" on https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv, and I will be updating the OPIC request doc to reflect the new name/acronym.

Marina said she'll update the OPIC.gov agency name to "United States International Development Finance Corporation" this coming week.

dav3r reviewed (commented):
@jsf9k I haven't looked thru the code, but I am wondering if we are going to have any issues if two agencies in this file have the same ID (`DFC`).

climber-girl committed:
Add AFRH

Adding the Armed Forces Retirement Home, since they were recently added to the FED_GOLD list.

climber-girl committed:
Add NCMNPS

Adding the National Commission on Military, National, and Public Service, since they were recently added to the FED_GOLD list.

climber-girl commented on include/agencies.csv:
```diff
@@ -83,7 +83,7 @@ Nuclear Regulatory Commission,NRC
 Office of Government Ethics,OGE
 Office of Personnel Management,OPM
-Overseas Private Investment Corporation,OPIC
+Overseas Private Investment Corporation,DFC
```
Can probably just remove line 86 since, after discussing with @jsf9k, we are going to wait for GSA to update the agency name on https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv.

mcdonnnj reviewed (changes requested):
Now that GSA has updated would you remove line 88 to eliminate the unneeded DFC mapping?

climber-girl committed:
Remove OPIC full name mapping

GSA has changed OPIC to DFC, so this line is no longer needed

jsf9k reviewed (approved):
:+1:

jsf9k commented:
Approved, but the version needs to be bumped.

mcdonnnj committed:
Bump version from 1.2.0 to 1.2.1

jsf9k commented:
> Approved, but the version needs to be bumped before merging. @climber-girl, can you open `version.txt` in a text editor and change the contents to `1.2.1`?

Nevermind, @mcdonnnj took care of it!

dav3r reviewed (approved):
LGTM! 🚀
